nixos/modules/services/matrix/dendrite.nix

```diff
@@ -196,7 +196,7 @@ in
           };
         };
         options.sync_api.search = {
-          enable = lib.mkEnableOption (lib.mdDoc "Dendrite's full-text search engine");
+          enabled = lib.mkEnableOption (lib.mdDoc "Dendrite's full-text search engine");
           index_path = lib.mkOption {
             type = lib.types.str;
             default = "${workingDir}/searchindex";
```

Comment:
Do you see value in having this value for `index_path` being set? Or has it a good value already set by default?

Reply:
Looking at it again, the default within Dendrite works fine for us because we set the working directory appropriately, so the default without this option definition is the same that it currently is. (Default in dendrite is `./searchindex`.)

I'm fine with dropping the search section entirely if that would be best, since all the defaults work for us.